tests/swarm-test.js

```diff
@@ -24,6 +24,16 @@ process.on('uncaughtException', function (err) {
   console.log('Caught exception: ' + err)
 })
 
+experiment('Without a peer', function () {
+  test('it throws an exception', function (done) {
+    expect(function () {
+      var sw = new Swarm()
+      sw.close()
```

Comment:
had to do something with it because annoying `standard` was complaining on "Do not use 'new' for side effects"